opentracing-spring-cloud/src/test/java/io/opentracing/contrib/spring/cloud/MockTracingConfiguration.java

```diff
@@ -42,4 +44,17 @@ public RestTemplate restTemplate(RestTemplateBuilder restTemplateBuilder) {
   public AsyncRestTemplate asyncRestTemplate() {
     return new AsyncRestTemplate();
   }
+
+  @RestController
+  public static class TestController {
```

Comment:
I think It make more sense to have it in a separate file. It's static and it has to be added to test configuration classes explicitly.